packages/backend/src/logger.ts

```diff
@@ -0,0 +1,69 @@
+import winston from 'winston';
+
+const levels = {
+    error: 0,
+    warn: 1,
+    info: 2,
+    http: 3,
+    debug: 4,
+};
+
+const level = () => {
+    if (process.env.LIGHTDASH_LOG_LEVEL) {
+        return process.env.LIGHTDASH_LOG_LEVEL.toLowerCase();
+    }
+    const env = process.env.NODE_ENV || 'development';
+    const isDevelopment = env === 'development';
+    return isDevelopment ? 'debug' : 'warn';
+};
+
+const colors = {
+    error: 'red',
+    warn: 'yellow',
+    info: 'green',
+    http: 'magenta',
+    debug: 'white',
+};
+
+winston.addColors(colors);
+
+const terminalFormat = winston.format.combine(
+    winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+    winston.format.colorize({ all: true }),
+    winston.format.printf(
+        (info) =>
+            `${info.timestamp} [Lightdash] ${info.level}: ${info.message}`,
+    ),
+);
+
+const jsonFormat = winston.format.combine(
+    winston.format.timestamp(),
+    winston.format.json(),
+);
+
+const transports = [
```

Comment:
Nice setup, what's the purpose for multiple log formats written to files?

Reply:
`.log` is human readable
`.json.log` is to import in log tools/services

Reply:
I'm going to change the error.json.log to error.log and have a terminal formatting since it is more usuful for humans. 
If people want to use a tool they can upload the all.json.log and filter by level.